src/utils/hooks/useSubmitApplication.tsx

```diff
@@ -15,7 +15,7 @@ const useSubmitApplication = ({ serialNumber }: UseGetApplicationProps) => {
 
   const submit = async (structure: FullStructure) => {
     const elements = Object.values(structure.elementsById || {}).filter(
-      (element) => element.element.category === TemplateElementCategory.Question
+      (element) => !!element.latestApplicationResponse.id
```

Comment:
Needed to change this to `!!element.latestApplicationResponse?.id` to fix error occurring when submitting application